osgar/ro2018.py

```diff
@@ -192,19 +193,22 @@ def publish(self, channel, data):
     subparsers = parser.add_subparsers(help='sub-command help', dest='command')
     subparsers.required = True
     parser_run = subparsers.add_parser('run', help='run on real HW')
-    parser_run.add_argument('config', help='configuration file')
+    parser_run.add_argument('config', nargs='+', help='configuration file')
     parser_run.add_argument('--note', help='add description')
 
     parser_replay = subparsers.add_parser('replay', help='replay from logfile')
     parser_replay.add_argument('logfile', help='recorded log file')
     parser_replay.add_argument('--force', '-F', dest='force', action='store_true', help='force replay even for failing output asserts')
+    parser_replay.add_argument('--config', nargs='+', help='force alternative configuration file')
     args = parser.parse_args()
 
     if args.command == 'replay':
         log = LogReader(args.logfile)
         print(next(log.read_gen(0))[-1])  # old arguments
         config_str = next(log.read_gen(0))[-1]
         config = literal_eval(config_str.decode('ascii'))
+        if args.config is not None:
+            config = Config.load(args.config).data
```

Comment:
The same problem as above.

Reply:
Maybe we could do without any class. Just have a function in `osgar/lib/config.py` with signature `load(*filenames)` and let it return what is now stored in `Config.data`. AFAIK `data` is the only thing in the class anyway, right?

Reply:
As we use it now, it is only the dictionary ... and when we will need more, we can change it in the future. I also do not like much addressing via `Config.data`.